<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Ashwini" failures="1" tests="4" name="corporateModule.CorporateCreatedByHrHead_BaseClass" time="1.460" errors="0" timestamp="2023-03-29T10:05:32 IST" skipped="3">
  <testcase classname="corporateModule.CorporateCreatedByHrHead_BaseClass" name="CorporateCreatedByHrHead" time="1.036">
    <skipped/>
  </testcase> <!-- CorporateCreatedByHrHead -->
  <system-out>
    <![CDATA[CorporateCreatedByHrHead---> TestScript excecution started
TestScript Excecution Skipped]]>
  </system-out>
  <testcase classname="corporateModule.CorporateCreatedByHrHead_BaseClass" name="CorporateCreatedByHrHead" time="0.170">
    <skipped/>
  </testcase> <!-- CorporateCreatedByHrHead -->
  <system-out>
    <![CDATA[CorporateCreatedByHrHead---> TestScript excecution started
TestScript Excecution Skipped]]>
  </system-out>
  <testcase classname="corporateModule.CorporateCreatedByHrHead_BaseClass" name="CorporateCreatedByHrHead" time="0.117">
    <failure message="expected [Login Page is not displayed and it is verified ] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Login Page is not displayed and it is verified ] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:617)
at corporateModule.CorporateCreatedByHrHead_BaseClass.CorporateCreatedByHrHead(CorporateCreatedByHrHead_BaseClass.java:22)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- CorporateCreatedByHrHead -->
  <system-out>
    <![CDATA[CorporateCreatedByHrHead---> TestScript excecution started
TestScripts Exc Failed]]>
  </system-out>
  <testcase classname="corporateModule.CorporateCreatedByHrHead_BaseClass" name="CorporateCreatedByHrHead" time="0.137">
    <skipped/>
  </testcase> <!-- CorporateCreatedByHrHead -->
  <system-out>
    <![CDATA[CorporateCreatedByHrHead---> TestScript excecution started
TestScript Excecution Skipped]]>
  </system-out>
</testsuite> <!-- corporateModule.CorporateCreatedByHrHead_BaseClass -->
